### ISSUE TITLE
PHP gd pdo_mysql and soap extensions missing on Fedora

### ISSUE THREAD

**Issue by Jelmerb** [tdgroot/ansible-workstation-common#71]
When I install a fresh copy of Magento 2 using composer I get a message that the following PHP extensions are not installed
* `gd`
* `pdo_mysql`
* `soap`

Are these already installed and simple have to be enabled? Otherwise, it might be good to install them along with PHP. 
The packages are called `php-gd` `php-mysqlnd` and `php-soap`.

**Comment by tdgroot** [comment]
Try to load php 7.4 with `module load php74`, does that solve it?

**Comment by Jelmerb** [comment]
Yes it does. Thanks!

**Comment by tdgroot** [comment]
@Jelmerb it helps to add the bin directory of a specific PHP version to your PATH:
``` shell
echo export PATH=/opt/remi/php74/root/bin:$PATH >> ~/.zshrc

source ~/.zshrc # Reload zshrc file
```

**Comment by Jelmerb** [comment]
Because the system already used PHP 7.14.16, I thought that the system used the PHP installation from your playbook, but Fedora probably has its own version of PHP installed. Should I add more directories to my PATH?

**Comment by tdgroot** [comment]
@Jelmerb exactly. Composer is installed system-wide and requires the php-cli package, which unfortunately doesn't get resolved to any php{7.3, 7.4, 8.0}-cli version.

> Should I add more directories to my PATH
I'd leave it like this for now!
